string face_type
---
bool response
